app/Models/Competition.php

```diff
@@ -12,14 +11,14 @@ class Competition extends Model
      *
      * @var array
      */
-    protected $fillable = ['campaign_id', 'campaign_run_id', 'start_date', 'end_date'];
+    protected $fillable = ['contest_id', 'competition_start_date', 'competition_end_date'];
```

Comment:
I believe, yes, gladiator creates the competitions automatically when the waiting rooms get split ( @deadlybutter is that correct?) But even so, doesn't it need to be able to be mass assigned?